topics/variant-analysis/tutorials/trio-analysis/tutorial.md

```diff
@@ -0,0 +1,438 @@
+> > > 4. Give the new collection a name and click on **Create collection**.
+> > >
+> > {: .tip}
+> >
+> {: .hands_on}
+> 
+{: .comment}
+
+From here you can go to [pre-processing](#pre-processing) if you donwloaded the files from zenodo, otherwise continue with the tutorial from here.
+
+## Getting DAC access
+Our test data is stored in EGA, which can be easily accessed using the EGA Download Client. Our specific EGA dataset accession ID is: "EGAD00001008392". However, before you can access this data you need to request DAC access to this dataset. This can be requested by emailing to <helpdesk@ega-archive.org>, don’t forget to mention the dataset ID! When the EGA grants you access it will create an account for you, if you don't have it already. Next, you should link your account to your galaxy account by going to the homepage on galaxy and at the top bar click **User > Preferences > Manage Information**. Now add your email and password of your (new) EGA account under: **Your EGA (european Genome Archive) account**. After that you can check if you can log in and see if you have access to the dataset. 
+
+> ### {% icon hands_on %} Hands-on: Check log-in and authorized datasets
+>
+> 1. {% tool [EGA Download Client](toolshed.g2.bx.psu.edu/repos/iuc/ega_download_client/pyega3/4.0.0+galaxy0) %} with the following parameters:
+>    - *"What would you like to do?"*: `List my authorized datasets`
+>
+> > ### {% icon comment %} Comment: Check if the dataset is listed.
+> > 
+> > Check if your dataset is listed in the output of the tool. If not you can look at the header of the output to find out why it is not listed. When the header does not provide any information you could have a look at the error message by clicking on the **eye** {% icon galaxy-eye %} of the output dataset and then click on the icon **view details** {% icon details %}. The error should be listed at **Tool Standard Error**.
+> {: .comment}
+>
+{: .hands_on}
+
+## Download list of files
+When you have access to the EGA dataset, you can download all the needed files. However, the EGA dataset contains many different filetypes and cases, but we are only interested in the VCFs from case 5 and, to reduce execution time, the variants on chromosome 17. To be able to donwload these files we first need to request the list of files from which we can download. Make sure to use **version 4+** of the {% tool [EGA Download Client](toolshed.g2.bx.psu.edu/repos/iuc/ega_download_client/pyega3/4.0.0+galaxy0) %}.
+
+> ### {% icon hands_on %} Hands-on: Request list of files in the dataset
+>
+> 1. {% tool [EGA Download Client](toolshed.g2.bx.psu.edu/repos/iuc/ega_download_client/pyega3/4.0.0+galaxy0) %} **version 4+** with the following parameters:
+>    - *"What would you like to do?"*: `List files in a datasets`
+>    - *"EGA Dataset Accession ID?"*: `EGAD00001008392`
+>
+> > ### {% icon tip %} Tip: Changing tool version.

```

Comment:
There is also already a snippet for this (https://training.galaxyproject.org/training-material/faqs/galaxy/#tools)

Reply:
FIxed. b513e57556134e26bf9dab7905a00d75fb516bc4